platforms/core-runtime/build-operations/src/main/java/org/gradle/internal/operations/BuildOperationProgressEventEmitter.java

```diff
@@ -26,8 +26,9 @@
  */
 @ServiceScope(Global.class)
 public interface BuildOperationProgressEventEmitter {
-    void emit(OperationIdentifier operationIdentifier, long timestamp, @Nullable Object details);
+    void emit(OperationIdentifier operationIdentifier, long timestamp, Object details);
```

Comment:
🤔 did the nullability change here? I assume this method is also used somewhere else.

Reply:
Actually, I do not remember why I changed this. 
I will revert this, as I believe I did something unnecessary here.